### ISSUE TITLE
Deploy

### ISSUE THREAD

**Issue by jcahail** [azavea/pfb-network-connectivity#666]
Final deploy for the contract tasks.

**Comment by KlaasH** [comment]
Special steps:
- Set `use_tilegarden` to `true` in the tfvars file
- Since deploying the new service definitions with that set will cause the site to switch to Tilegarden, we need to have Tilegarden updated before running `infra apply`
- As part of updating Tilegarden, we'll need to add S3 permissions to the `pfbTilegardenProduction-executor` role
- After we've applied the service definition changes but before the new instances of the HTTPS service spin up, we need to run migrations

**Comment by KlaasH** [comment]
A few notes on how this ended up going:
- It turned out this was a 3-step rather than a 2-step thing: first do the full Terraform deploy with `use_tilegarden = false` to get the cache bucket set up; then deploy Tilegarden with the cache bucket added to `.env` and test out a tile; then set `use_tilegarden = true` and do a full deploy (re-deploying Tilegarden again was redundant, but better to use `infra apply` than do it piecemeal)
- Running a migration task after the service definitions were updated but before any new tasks spun up turned out not to work.  It failed to run due to not having an instance available with enough memory.  So instead I logged onto one of the instances and got a shell in one of the new containers moments after it came up.  I think that's before traffic was being directed to it, though I don't know exactly when during the period that both old and new instances are up the new instances start getting requests.  In any case, this time I believe it was fine, but issue #664 is still relevant.
